blog/tests.py

```diff
@@ -118,3 +132,172 @@ def test_delete_blog(self):
         force_authenticate(request, user=user)
         response = view(request, slug=blog.slug)
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+    def test_update_blog(self):
+        """
+        Tests to update a Blog Post Successfully.
+        """
+        response = self.create_blog()
+        user = User.objects.get(username='test')
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        blog = Blog.objects.get()
+
+        body = {
```

Comment:
Always use verbose names. Sometimes we need to think hard for a good name.
We can rename it as `blog_update_content`

Reply:
Renamed them all.